staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -302,6 +355,19 @@ func (s *APIAggregator) AddAPIService(apiService *apiregistration.APIService) {
 	s.handledGroups.Insert(apiService.Spec.Group)
 }
 
+func (s *APIAggregator) deferLoadAPISpec(name string) {
```

Comment:
Do I get this right that this only puts the name onto the radar for the next update call which is triggered through a OpenAPI download request at https://github.com/kubernetes/kubernetes/pull/46734/files#diff-50c8cb84572c056e12063729607ca9edR149 ? If so, if we want such a interaction complexity at all,  this deserves a decent comment. Where is this tested that we don't break that link?